fuzz/meson.build

```diff
@@ -7,12 +7,14 @@ fuzzers = [
   'bundle',
 ]
 
+extra_c_args = cc.get_supported_arguments('-Wno-missing-prototypes')
```

Comment:
This name is quite generic and could be set by another meson.build file, too (like `extra_test_sources` already is) Maybe use something like `fuzz_c_args=`?